extensions/cornerstone/src/init.tsx

```diff
@@ -41,7 +40,6 @@ export default async function init({
   servicesManager,
   commandsManager,
   extensionManager,
```

Comment:
Can we make the name of the file/function consistent - it is preRegistration in the extension vocabulary.  Since you are here and changing it anyways.

Reply:
There are 4 other init ones

![CleanShot 2024-03-12 at 13 41 08](https://github.com/OHIF/Viewers/assets/7490180/75e5885f-36b5-40fe-8336-010db60dca5a)

I rather do it in another PR really just to be clear about the PR commit message